crates/ra_ide_api/src/completion/complete_postfix.rs

```diff
@@ -21,28 +22,46 @@ fn postfix_snippet(ctx: &CompletionContext, label: &str, detail: &str, snippet:
         .snippet_edit(edit)
 }
 
+fn is_bool_or_unknown(ctx: &CompletionContext) -> bool {
```

Comment:
it would make more sense to me to get the receiver type already in `complete_postfix` and have this function just take a `Ty` (or `Option<Ty>`)... we'll have more postfix completions that want to check types in the future :slightly_smiling_face: